providers/implementations/ciphers/ciphercommon_gcm.c

```diff
@@ -51,7 +50,7 @@ static int gcm_init(void *vctx, const unsigned char *key, size_t keylen,
     ctx->enc = enc;
 
     if (iv != NULL) {
-        if (ivlen < ctx->ivlen_min || ivlen > sizeof(ctx->iv)) {
+        if (ivlen > sizeof(ctx->iv)) {
```

Comment:
Should it be checking ivlen == 0 here?

Reply:
Would a zero IV be even better than a one byte one?
I wonder if the code even works with a zero IV....it fails much later.  I'll add the check here too.